### PR TITLE
Consumed the mark question and assign bonus api

### PR DESCRIPTION
**Pull request by silasade**

### CODE REVIEW

File: src/Global Components/SchoolDetails/SchoolDetails.tsx
Comment (a-k-o-r-e-d-e) [inline src/Global Components/SchoolDetails/SchoolDetails.tsx:1]: line 118 & 179, use the base url already declared

File: src/Global Components/Types/Types.tsx
Comment (a-k-o-r-e-d-e) [inline src/Global Components/Types/Types.tsx:1]: Probably not the best way to declare the types, but i will allow

File: src/features/getQuestionID.tsx
Comment (a-k-o-r-e-d-e) [inline src/features/getQuestionID.tsx:1]: why do you need a slicer to get question ID, just curious

**Review by a-k-o-r-e-d-e — CHANGES_REQUESTED** [review]

File: src/Global Components/SchoolDetails/SchoolDetails.tsx
Comment (silasade) [reply src/Global Components/SchoolDetails/SchoolDetails.tsx:1]: This issue has been resolved.

File: src/features/getQuestionID.tsx
Comment (silasade) [reply src/features/getQuestionID.tsx:1]: I had to make sure the question id is visible in the parent component, I am open to better ways of solving it.

File: src/Global Components/Types/Types.tsx
Comment (silasade) [reply src/Global Components/Types/Types.tsx:1]: Already resolved it

**Review by a-k-o-r-e-d-e — APPROVED** [review]

File: src/features/getQuestionID.tsx
Comment (a-k-o-r-e-d-e) [reply src/features/getQuestionID.tsx:1]: There should be a simpler way of achieving the same thing but for the sake of speed. I will allow for now

**Pull request merged**